src/containers/homepage/homepage.container.less

```diff
@@ -1,5 +1,6 @@
 .layout {
-  height: 100%;
+  display: flex;
+  min-height: 100vh;
```

Comment:
I'm never 100% confident in seeing this - as soon as any change occurs where there is anything displayed outside of the `HomepageContainer` (like a nav/footer that's 1 level higher in the dom) this could mess with the page

Reply:
I wholeheartedly agree. It seems I fixed this bug on the wrong branch as there is no AdminLayoutContainer on this branch and that is where it should go.

Reply:
I will pull upstream changes and apply this styling to AdminLayoutContainer.